tests/scripts/test-ref-configs.pl

```diff
@@ -32,11 +32,19 @@
         'compat' => '-m tls12 -f \'^TLS-PSK-WITH-AES-...-CCM-8\'',
         'test_again_with_use_psa' => 1
     },
+    'config-ccm-psk-dtls1_2.h' => {
+        'compat' => '-m dtls12 -f \'^TLS-PSK-WITH-AES-...-CCM-8\'',
```

Comment:
This actually runs no `compat.sh` test cases. This is a preexisting bug (https://github.com/ARMmbed/mbedtls/issues/5629) and out of scope here.

Reply:
https://jenkins-mbedtls.oss.arm.com/blue/rest/organizations/jenkins/pipelines/mbed-tls-pr-merge/branches/PR-5582-merge/runs/22/nodes/282/steps/1009/log/?start=0 (CI run with #5630 merged) shows that `compat.sh` runs some tests here:
```
m->G dtls12,no TLS-PSK-WITH-AES-128-CCM-8 .............................. PASS
m->G dtls12,no TLS-PSK-WITH-AES-256-CCM-8 .............................. PASS
m->m dtls12,no TLS-PSK-WITH-AES-128-CCM-8 .............................. PASS
m->m dtls12,no TLS-PSK-WITH-AES-256-CCM-8 .............................. PASS
m->G dtls12,yes TLS-PSK-WITH-AES-128-CCM-8 ............................. PASS
m->G dtls12,yes TLS-PSK-WITH-AES-256-CCM-8 ............................. PASS
m->m dtls12,yes TLS-PSK-WITH-AES-128-CCM-8 ............................. PASS
m->m dtls12,yes TLS-PSK-WITH-AES-256-CCM-8 ............................. PASS
```
This looks as expected (we're testing against OpenSSL 1.0.2g and it doesn't support CCM cipher suites, hence no testing against OpenSSL).